source/admin_manual/mailbox_formats/pop3c.rst

```diff
@@ -0,0 +1,12 @@
+.. _pop3c_mbox_format:
+
+====================
+Pop3c Mailbox Format
+====================
+
+The pop3c storage accesses a remote POP3 server as if it were a regular
+(local) Dovecot mailbox format. Dovecot can treat it as a dummy storage or
+optionally a more capable storage.
```

Comment:
I see this is just copied from imapc, but I don't see how pop3c could be treated as anything else than a dummy storage. :) So just remove this sentence?

Also could say that the the POP3 mailbox is visible as the INBOX folder on Dovecot side.